src/VisualStudio/Core/Test/ServicesVisualStudioTest.vbproj

```diff
@@ -146,16 +143,16 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="WindowsBase" />
   </ItemGroup>
-  <Target Name="AddShellImmutable15" Condition="'$(VisualStudioVersion)' == '15.0'" BeforeTargets="ResolveAssemblyReferences">
-    <!-- we want this to only be added for Dev15 builds, but the project system doesn't like us having conditional
+  <!--<Target Name="AddShellImmutable15" Condition="'$(VisualStudioVersion)' == '15.0'" BeforeTargets="ResolveAssemblyReferences">
```

Comment:
IIRC this was only required since we weren't on the NuGet packages that were actually stable. This can probably be actually deleted now?